tools/loader/ImageClassifier.cpp

```diff
@@ -671,12 +677,12 @@ int main(int argc, char **argv) {
         timer.startTimer();
       }
       unsigned requestCount = miniBatch ? iterationsOpt / miniBatch : 1;
-      runBenchmark(name, loader, std::move(contexts), requestCount);
+      runBenchmark(name, loader, std::move(contexts), requestCount, warmup);
       if (timeOpt) {
         timer.stopTimer();
-        llvm::outs() << llvm::formatv("Wall time per item (s): {0:f4}\n",
-                                      timer.getTotalTime().getWallTime() /
-                                          iterationsOpt);
+        llvm::outs() << llvm::formatv(
+            "Wall time per item (s): {0:f4}\n",
+            timer.getTotalTime().getWallTime() / iterationsOpt + warmup);
```

Comment:
this does not seem correct: timer.getTotalTime().getWallTime() / iterationsOpt + warmup => timer.getTotalTime().getWallTime() / (iterationsOpt + warmup)

Reply:
Uff, thanks for catching it